### PR TITLE
Hdxdsys 1087

### PR DESCRIPTION
**Pull request by davidmegginson**
Updated documentation for new humanitarian needs source (HPC Tools) and to point to Erica's data-availability viz instead of a static table.

HDXDSYS-1087

### CODE REVIEW

File: docs/data_usage_guides/endpoint_parameters/humanitarian_needs_parameters.yaml
Comment (mcarans) [inline docs/data_usage_guides/endpoint_parameters/humanitarian_needs_parameters.yaml:39]: Just confirming that it's intentional that the freeform category column cannot be passed as a parameter as I don't see it here?

**Review by mcarans — APPROVED** [review]
Thanks for this update.

File: docs/data_usage_guides/endpoint_parameters/humanitarian_needs_parameters.yaml
Comment (alexandru-m-g) [reply docs/data_usage_guides/endpoint_parameters/humanitarian_needs_parameters.yaml:39]: @davidmegginson @mcarans this is related to the free form category for humanitarian needs, right ? We should only merge this, once the code for this is also merged in HAPI.

**Comment by davidmegginson** [comment]
@alexandru-m-g @turnerm  could we merge this into a dev branch for now?

**Review by alexandru-m-g — APPROVED** [review]

**Pull request merged**
